src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import static core.basesyntax.Application.MAX_POSSIBLE_NUMBER_FOR_BALL;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(),
+                random.nextInt(MAX_POSSIBLE_NUMBER_FOR_BALL));
+    }
+}

```

Comment:
You are using this constant only here, so make it class-related constant. 
```suggestion
import java.util.Random;

public class Lottery {
private static final int MAX_POSSIBLE_NUMBER_FOR_BALL = ...;

    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(),
                random.nextInt(MAX_POSSIBLE_NUMBER_FOR_BALL));
    }
}

```